server/channels/store/sqlstore/migrate.go

```diff
@@ -95,7 +96,7 @@ func (m *Migrator) GetFileName(plan *models.Plan) (string, error) {
 	return fmt.Sprintf("migration_plan_%d_%d", from, to), nil
 }
 
-func (ss *SqlStore) initMorph(dryRun bool) (*morph.Morph, error) {
+func (ss *SqlStore) initMorph(dryRun, disableLogging bool) (*morph.Morph, error) {
```

Comment:
Can we invert the flag to check for positive case (enableLogging) rather than the negative case (disableLogging)? It is slightly easier to think false means "disable something". :)

Reply:
Indeed, I see that now: if this were an omittable struct field, the inverted semantics would be better, but agree this is harder to use.